packages/OreIdProfile/src/OreIdProfileButton/OreIdProfileButton.tsx

```diff
@@ -55,7 +55,7 @@ export const OreIdProfileButton: React.FC<Props> = ({
 									})}
 									style={{ color: style?.textColor || "#222222" }}
 								>
-									<Icon icon={userData.picture.href} size={32} />{" "}
```

Comment:
Did you test this? when I tested I had to put it like this to work

Reply:
the version of oreid-js with this has the correct fix for this issue